full_rebuild.rst

```diff
@@ -0,0 +1,31 @@
+Howto: Perform a full rebuild
```

Comment:
Maybe this should be added to the [wiki](https://github.com/mtgjson/mtgjson/wiki) instead of the repo files?